content/automate/ManageIQ/System/CommonMethods/QuotaMethods.class/__methods__/requested.rb

```diff
@@ -165,8 +165,8 @@ def requested_storage(args_hash)
     if args_hash[:resource].options[:disk_remove]
       args_hash[:resource].options[:disk_remove].each do |disk|
         $evm.log(:info, "Reconfigure Disk Removal: #{disk.inspect}")
-        current_size = get_disk_size(disk['disk_name'])
-        $evm.log(:info, "Reconfigure Disk Removal Disk: <#{disk['disk_name']}> Disk Size: <#{current_size.to_s(:human_size)}>")
+        current_size = get_disk_size(disk[:disk_name])
```

Comment:
@billfitzgerald0120 
Should we try converting the disk to be hashWithIndifferntAccess
e.g.
```
x = {:a => 1, 'b' => 2
y = HashWithIndifferentAccess.new(x)
y[:b]
y['b']
```
In our case we could use
disk = HashWithIndifferentAccess.new(disk)

Reply:
@mkanoor Made changes as requested, please review